applications/asset_tracker/prj_nrf9160_pca20035ns.conf

```diff
@@ -26,6 +26,10 @@ CONFIG_POWER_OPTIMIZATION_ENABLE=n
 CONFIG_LTE_LINK_CONTROL=y
 CONFIG_LTE_AUTO_INIT_AND_CONNECT=n
 
+CONFIG_LTE_LOCK_BANDS=y
+# Enable band 2, 3, 4, 8, 12, 13 ,20 and 28
+CONFIG_LTE_LOCK_BAND_MASK="1000000010000001100010001110"
```

Comment:
I'm curious - why is it only applicable for pca20035 and not for pca10090?

Reply:
Because pca10090 supports all bands the modem supports and does not need to be limited.